src/legacy/core_plugins/tile_map/common/ems_client.js

```diff
@@ -92,8 +92,10 @@ const unescapeTemplateVars = url => {
 const DEFAULT_LANGUAGE = 'en';
 
 
+
 export class EMSClientV66 {
 
+  EMS_LOAD_TIMEOUT = 8000;
```

Comment:
honestly, I'd make it longer. gut says that this timeout might come bite us, just given the wide variety of environments kibana will be deployed (slow networks are more common than we'd think (?)). I'd make it something like 32s imho. but I do agree with the reasoning behind it.

Reply:
Okay, done!